src/Service/AlertManager.php

```diff
@@ -283,7 +283,7 @@ private function checkVisibility(NodeInterface $alert, NodeInterface $node) {
     $current_path = $current_path ? mb_strtolower($current_path) : $current_path;
     // Check all values from the field "alert_visibility_pages".
     foreach ($pages as $page) {
-      $page_path = $page === '<front>' ? '/' : '/' . ltrim($page, '/');
+      $page_path = $page === '<front>' ? '<front>' : '/' . ltrim($page, '/');
```

Comment:
```suggestion
      $page_path = ($page === '<front>' || $page === '/') ? '<front>' : '/' . ltrim($page, '/');
```

Let's account for the old version of home (`/`) too.